server/repository/src/db_diesel/changelog/changelog.rs

```diff
@@ -566,6 +598,50 @@ fn create_filtered_outgoing_sync_query(
     query
 }
 
+// This is a manual sync to fetch all records for a specific patient
+// Managed via own cursor
+fn create_filtered_outgoing_patient_sync_query(
+    earliest: u64,
+    sync_site_id: i32,
+    fetch_patient_id: String,
+) -> BoxedChangelogQuery {
+    let mut query = changelog_deduped::table
+        .left_join(name_link::table)
+        .filter(changelog_deduped::cursor.ge(earliest.try_into().unwrap_or(0)))
```

Comment:
I was trying to make a petter of these methods, but it hasn't fully taken off:

https://github.com/msupply-foundation/open-msupply/blob/98b80b3bf57e87ee199ac74b5f4bbe0d82d8ed2e/server/service/src/lib.rs#L289-L311

Reply:
ohh tbh this was copy and paste, didn't realise it was converting a type - I like these helpers! Can update